src/request.ts

```diff
@@ -33,13 +29,13 @@ function defaultDataExtractor(data: string): Location {
   };
 }
 
-export function requestLocatorFactory(): Function {
-  function resourceLocator(parameters: string): Locator.Locator;
-  function resourceLocator(parameters: RequestLocatorParameters): Locator.Locator;
-  function resourceLocator(parameters: any): Locator.Locator {
-    if (typeof parameters === "string") parameters = { url: parameters };
-    var url: string = parameters.url;
-    var request: any;
+export function requestLocatorFactory():Function {
+  function resourceLocator(parameters:string):Locator;
```

Comment:
I don't know, because there was no linter to tell me to do that? I'll add the linter to the compile process too...